layouts/shortcodes/featured/header.html

```diff
@@ -15,6 +15,11 @@
 {{ $focus := .Get "image-focus" }}
 {{ $gravity := .Get "image-gravity" }}
 {{ $artType := .Get "art-type" | default "Photo" }}
+{{ $artCredit := .Get "art-credit" }}
+{{ $artSource := .Get "art-source" }}
+{{ $artDesc := .Get "art-description" }}
+{{ $artType2 := .Get "art-type2" | default "Photo" }}
```

Comment:
It's a little confusing that this one is art-type2 but the credit is art-credit and art-type affects credit 1.

Reply:
I am not sure what the naming convention should be, since `art-type` already exists on other template stories and I can't use `video` without potentially breaking template stories with videos. should I use `image-*-2` even though it's a video and not technically an image that we're crediting?

Reply:
I would do second-credit, second-source, second-description, second-type.

Reply:
LMK if this looks good and I can merge: https://github.com/spotlightpa/poor-richard/pull/489/commits/043a3f64173850e56f8e26fc7bf841e229ba87d0

Reply:
I've gotta go ahead and merge for the story tomorrow but I can make more changes post-launch if needed